### PR TITLE
Add Infer Mitochondria Job

### PR DESCRIPTION
**Pull request by fm3**
Adds infer_mitochondria worker job (for now for superusers only)

Corresponding PR in voxelytics: https://github.com/scalableminds/voxelytics/pull/3504

### Steps to test:
- Set up worker with vx branch `infer-mitochondria-workflow`, in wk, run `yarn enable-jobs`, open dataset/annotation in wk, create a bbox with a mito, hit “AI Analysis”, run mito inferral job, result should look ok

------
- [x] (no changelog for now, since this is still behind the superUser condition)
- [x] Updated [migration guide](../blob/master/MIGRATIONS.unreleased.md#unreleased) if applicable
- [x] Removed dev-only changes like prints and application.conf edits
- [x] Considered [common edge cases](../blob/master/.github/common_edge_cases.md)

### CODE REVIEW

File: frontend/javascripts/oxalis/view/action-bar/starting_job_modals.tsx [hunk 1]
```diff
@@ -719,6 +722,58 @@ export function NeuronSegmentationForm() {
...
+          bbox,
+          outputSegmentationLayerName,
+          newDatasetName,
+        );
+      }}
+      description={
+        <>
+          <Space direction="vertical" size="middle">
+            <Row>
+              This job will automatically detect and segment all neurons in this dataset. The AI
```
Comment (hotzenklotz) [inline frontend/javascripts/oxalis/view/action-bar/starting_job_modals.tsx:759]:
```suggestion
              This job will automatically detect and segment all mitochondria in this dataset. The AI
```

File: frontend/javascripts/admin/dataset/dataset_upload_view.tsx [hunk 2]
```diff
@@ -108,7 +108,7 @@ function WkwExample() {
 
 function SingleLayerImageStackExample() {
   const description = `
-  great_dataset          # Root folder or zip archive (this outer container be omitted)
+  great_dataset          # Root folder or zip archive (this outer container may be omitted)
```
Comment (hotzenklotz) [inline frontend/javascripts/admin/dataset/dataset_upload_view.tsx:111]: :+1:

**Review by hotzenklotz — APPROVED** [review]
Looks good to me. 

Excited for the first mitos!

**Review by philippotto — APPROVED** [review]
looks good :) see my one comment about a copy&paste error, though.

**Pull request merged**